src/MoBi.UI/Views/EditContainerView.cs

```diff
@@ -144,8 +144,8 @@ private void initNameControl(ContainerDTO dto)
          var isInit = dto.Name.IsNullOrEmpty();
```

Comment:
I don't think we need this `isInit` flag. Looking at the before code, it looks wrong. We mark the edit as readonly = false, then enabled = false. 

It seems to be working because there's another method that enables the control `ContainerPropertiesEditable` after.

Reply:
In fact, let's remove this method and just use `ContainerPropertiesEditable`

Then in the presenter we can use the `_isNewEntity` flag and always explicitly enable or disable the naming and other controls